server/jetstream.go

```diff
@@ -1272,9 +1272,9 @@ func (jsa *jsAccount) checkBytesLimits(addBytes int64, storage StorageType) erro
 }
 
 func (jsa *jsAccount) acc() *Account {
-	jsa.mu.RLock()
+	//jsa.mu.RLock()
```

Comment:
What is that?

Reply:
Left over, removed lock since it is only set at creation and can't change. Will fix.